### DISCUSSION TITLE
Current bg level notification on WearOS

### DISCUSSION THREAD

**Discussion by usernamebeep** [NightscoutFoundation/xDrip#3076] [Ideas]
Right now I only get low alarm, high alarm or bg reading missed alert notification on my WearOS.
To get bg level, I need to install watch face but I would also like to get current bg level as notification everytime bg level is measured without installing watch face,

EDIT: Maybe it has something to do with the graph in push notification and my watch cannot render this notification?
Is it possible to remove graph from notification?

**Comment by tolot27** [comment]
@Navid200 Can we extend your table in #1762 to catch the wear apk, too?

**Comment by Navid200** [comment]
@tolot27 That table is about the basic xDrip alerts.
My hope is to address all of those items in that table myself.

This issue is about a watch.  

I don't have any experience with wear.  I will have no way of addressing this.

I am not sure how this fits in there.

**Comment by tolot27** [comment]
Many source files are duplicated for app and wear instead of shared. Hence, features exist for both app and wear but not consistently because wear code is not updated at the same time as the app code. Hence, having wear in the table or an identical table for wear would be awesome.

**Comment by usernamebeep** [comment]
Any updates on this? Still no way to remove graph?

**Comment by tolot27** [comment]
> Any updates on this? Still no way to remove graph?

Not yet. As you can see, this issue is not assigned to a developer or project plan.

What do you mean with "remove graph from notification"? Please can you explain it in more detail and add a screenshot, if suitable.

**Comment by usernamebeep** [comment]
There is graph with current bg and arrow under notifications on phone.

Maybe my Moto 360 2nd gen cannot render this graph (phone can and shows notification) and won't show notification at all.
Adding a checkbox to show/hide graph from notifications and just show bg level and trend might fix this problem but I'm not 100% sure that is the problem.

Do others get current bg level notification on watch?

**Comment by tolot27** [comment]
Does your watch run Wear OS or the "full" Android OS? Your suggestion does not apply to xDrip wear because notifications are totally different in Wear OS and there is no bg notification graph implemented in xDrip wear.

**Comment by usernamebeep** [comment]
I guess it has nothing to do with xDrip wear.
It is WearOS, it gets all the notifications the phone has - emails, chats, xdrip alerts, everything BUT not current bg level. It is the only notification missing from my watch.

**Comment by usernamebeep** [comment]
Do other watches receive all the xdrip push notifications from phone?

**Comment by Navid200** [comment]
Unfortunately, we have no active developer who is looking after the WearOS portion of xDrip.
This issue has been open for more than 2 years now.  Nothing is going to happen if a developer does not decide to support WearOS.

Even if an experienced in WearOS developer shows interest, my hope is that they will create a companion app for xDrip to address all WearOS functionality instead of the current situation. 

I am changing this issue to a discussion.  This conversation can continue as a discussion.  If some day we can resolve the dilemma and provide WearOS support, we can reopen an issue if needed.

**Comment by usernamebeep** [comment]
Maybe it helps if there was a toggle switch in xDrip to turn off graph from notification.
